packages/odyssey-storybook/src/components/Button/Button.stories.tsx

```diff
@@ -103,3 +104,35 @@ IconOnly.args = {
   icon: <SettingsIcon />,
   children: undefined,
 };
+
+export const Theme: Story<
+  ButtonProps & {
+    ColorBackgroundPrimary: string;
+    ColorBackgroundSecondary: string;
+  }
+> = (props) => {
+  const { ColorBackgroundPrimary, ColorBackgroundSecondary, ...rest } = props;
+  return (
+    <ThemeProvider
+      theme={{
+        [Button.theme]: { ColorBackgroundPrimary, ColorBackgroundSecondary },
+      }}
+    >
+      <Button {...rest} />
+    </ThemeProvider>
+  );
+};
+Theme.argTypes = {
+  ColorBackgroundPrimary: {
+    name: "primary background",
+    defaultValue: "#1662dd",
```

Comment:
I'm surprised that we're defining hex codes inside components...

Shouldn't the Button `ColorBackgroundPrimary` default value be something like `ColorPrimaryBase`, so that it inherits from the global token?

Reply:
@alexdahl-okta this is just demo code I included to show the theming working for those who are curious to play with it. You're correct we'd want to reference a variable like `ColorPrimaryBase` if we ever merged a real example like this I was being a bit lazy with the demo code. In the future exposing all the themeable hooks for every component will require real documentation effort something we probably won't have ready by end of Jan but can work toward